modules/nw-pod-network-connectivity-configuration.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * networking/verifying-connectivity-endpoint.adoc
+
+[id="nw-pod-network-connectivity-configuration_{context}"]
+= Configuring pod connectivity check placement
+
+As a cluster administrator, you can configure on which nodes the connectivity check pods run by modifying the `network.config.openshift.io` object named `cluster`.
+
+.Prerequisites
+
+* Install the {oc-first}.
+
+.Procedure
+
+. To edit the connectivity check configuration, enter the following command:
++
+[source,terminal]
+----
+$ oc edit network.config.openshift.io cluster
+----
+
+. In the text editor, update the `networkDiagnostics` stanza to specify the node selectors that you want for the source and target pods.
+
+. To commit your changes, save your changes and exit the text editor.
+
+.Verification
+
+To verify that the source and target pods are running on the intended nodes, enter the following command:
+
+[source,terminal]
+----
+$ oc get pods -n openshift-network-diagnostics -o wide
+----
+
+.Example output
+[source,text]
+----
+NAME                                    READY   STATUS    RESTARTS   AGE   IP            NODE                                                             NOMINATED NODE   READINESS GATES
+network-check-source-7cdbc9858d-77bsv   1/1     Running   0          19m   10.128.2.12   pdiak-05-27-2024-xt8qh-worker-b-5f99q                            <none>           <none>
```

Comment:
NIT: Can we alter the node names so the are more generic and do not use `master`?
For example:
```
oc get po -n openshift-network-diagnostics -o wide                                                                                                                                                                                                                                                                                           
NAME                                    READY   STATUS    RESTARTS   AGE     IP           NODE                                        NOMINATED NODE   READINESS GATES
network-check-source-84c69dbd6b-p8f7n   1/1     Running   0          9h      10.131.0.8   ip-10-0-40-197.us-east-2.compute.internal   <none>           <none>
network-check-target-46pct              1/1     Running   0          9h      10.131.0.6   ip-10-0-40-197.us-east-2.compute.internal   <none>           <none>
network-check-target-8kwgf              1/1     Running   0          9h      10.128.2.4   ip-10-0-95-74.us-east-2.compute.internal    <none>           <none>
network-check-target-jc6n7              1/1     Running   0          9h      10.129.2.4   ip-10-0-21-151.us-east-2.compute.internal   <none>           <none>
network-check-target-lvwnn              1/1     Running   0          9h      10.128.0.7   ip-10-0-17-129.us-east-2.compute.internal   <none>           <none>
network-check-target-nslvj              1/1     Running   0          9h      10.130.0.7   ip-10-0-89-148.us-east-2.compute.internal   <none>           <none>
network-check-target-z2sfx              1/1     Running   0          9h      10.129.0.4   ip-10-0-60-253.us-east-2.compute.internal   <none>           <none>

```